prow/spyglass/viewers/viewers.go

```diff
@@ -146,7 +146,6 @@ func RegisterViewer(viewerName string, metadata ViewMetadata, handler ViewHandle
 	}
 	viewHandlerReg.reg[viewerName] = handler
 	viewMetadataReg.reg[viewerName] = metadata
-	logrus.Infof("Spyglass registered viewer %s with title %s.", viewerName, metadata.Title)
```

Comment:
I think these logs are pretty useful: they're one-offs on launch, and I believe are also the only way to know that your viewer was registered if it fails during startup.